docs/plugins.rst

```diff
@@ -58,6 +59,10 @@ The temporary directory for the Girder Worker task is mapped into the running co
 under the directory ``/data``, so any files that were fetched into that temp directory
 will be available inside the running container at that path.
 
+By default, the image you specify will be pulled using the ``docker pull`` command.
+In some cases, you may not want to perform a pull, and instead want to rely on the
+image already being present on the worker system. If so, set ``pull_image`` to false.
```

Comment:
Can we specify that disabling pulling is interesting only if only the image name of `image:tag` (e.g `foo:latest`) is specified. In that case, it will prevent systematic update to the latest version. That said, in the scientific context, explicitly specifying the digest of the image is recommended. It ensures the execution context is always the same. See https://docs.docker.com/engine/reference/run/#image-digest